testgridshot

```diff
@@ -28,9 +28,8 @@
 #+   Configuration can be passed in via the environment. The following
 #+   variables are available:
 #+
-#+   UPLOAD_KEY: [*mandatory*]
-#+     The user key for 'vgy.me'. You need to create an account at 'vgy.me'
-#+      (https://vgy.me/auth/register) and a user key to be able to upload images.
+#+   GITHUB_TOKEN: [*mandatory*]
+#+     This token is used to (pull from and) push the images to the wiki repo.
```

Comment:
Can we add a note about the necessary oauth scope of the token?

I assume that we do not run into permission problems here, since the user already has write access to the wiki during release cut, right?

Reply:
Def. good point on documenting which scopes the token needs. Will add once we've sorted out where we want to host the images.